app/models/page.rb

```diff
@@ -9,7 +9,7 @@ class Page < ApplicationRecord
   has_many :goto_conditions, class_name: "Condition", foreign_key: "goto_page_id", dependent: :nullify
   acts_as_list scope: :form
 
-  ANSWER_TYPES = %w[number address date email national_insurance_number phone_number selection organisation_name text name].freeze
+  ANSWER_TYPES = %w[number address date email national_insurance_number phone_number selection organisation_name text name payment_link].freeze
```

Comment:
why is this an answer type? I thought it would have just been another attribute on the form model like "what happens next", declaration and support details.

Reply:
I've literally no idea. Have been chatting to Sam about it (not freezing you out, but we need to get used to you not being around to answer questions).

Reply:
💯 no probs, just thought i would ask the question. 😄 👍